openpype/hosts/photoshop/api/extension/host/index.jsx

```diff
@@ -263,6 +263,42 @@ function getActiveDocumentFullName(){
     };
 }
 
+function getActiveDocumentFormatResolution(){
+    /**
+     *   Returns a dict with the resolution dimenssion of the current doc
+     * */
+    if (documents.length == 0){
+        return null;
+    }
+
+    try {
+        savedUnit = app.preferences.rulerUnits;
+        app.preferences.rulerUnits = Units.PIXELS;
+        resDict = {"width" :app.activeDocument.width.value,
+                    "height": app.activeDocument.height.value}
+        app.preferences.rulerUnits = savedUnit;
+        return(JSON.stringify(resDict));
+    }catch(e){
+        // No doc is active.
+        return
```

Comment:
Based on my comment with the `json.loads(res)` on the `ws_stub` file, this should probably be `return {}`

Reply:
corrigé